src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/AuthService.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.ForbiddenException;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.UnauthenticatedException;
+
+import java.util.List;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+@Service
+public class AuthService {
```

Comment:
I wander if we couldn't move it to div-common-lib and re-use both in FR and Div?